### ISSUE TITLE
init.py subscripts output shouldn't be visible

### ISSUE THREAD

**Issue by frafra** [NINAnor/python-template#8]
This happens when generating a new project:

```
[...]
trim trailing whitespace.................................................Failed
- hook id: trailing-whitespace
- exit code: 1
- files were modified by this hook

Fixing .pre-commit-config.yaml

fix end of files.........................................................Failed
- hook id: end-of-file-fixer
- exit code: 1
- files were modified by this hook

Fixing .editorconfig
Fixing .gitignore
Fixing pyproject.toml
[...]
Pretty format TOML.......................................................Failed
- hook id: pretty-format-toml
- exit code: 1
- files were modified by this hook

File pyproject.toml is not pretty-formatted
Fixing file pyproject.toml
[...]
```

**Comment by nicokant** [comment]
Generated files cannot be pretty, since they are generated by a jinja
template.
That's why precommit is run after cloning the template.

In order to get pretty formatted code from jinja the templates would need
to be unreadable and be totally dependant on whitespaces, empty lines and
end of lines.

Of course the template should try to fix known errors reported by the
linter, but a prettify step will always be required.

What's important is that the committed code is actually pretty.

**Comment by frafra** [comment]
If that is the normal behaviour, the user should not see the failure messages.

![task-failed-successfully-windows-xp-dialog-box-3574269544](https://github.com/NINAnor/python-template/assets/4068/b2be31aa-225f-4c9e-85f0-2a289ad6f427)

**Comment by nicokant** [comment]
resolved by 0573c2a114c95a632940720c258eacb4ac22f57d
